app/components/satis/dropdown/component_controller.js

```diff
@@ -875,25 +876,33 @@ export default class extends ApplicationController {
     if(this.isMultipleValue
       || this.freeTextValue
       || this.hiddenSelectTarget.options.length === 0
-    ) return false;
+      || this.element.querySelector(":focus") === null
+  ) return false;
 
       Array.from(this.hiddenSelectTarget.options).forEach((option) => {
         let item = this.itemsTarget.querySelector(`[data-satis-dropdown-item-value="${option.value}"]`)
         if (!item) {
           item = document.createElement("div")
-          item.setAttribute("data-satis-dropdown-target", "item")
-          item.setAttribute("data-action", "click->satis-dropdown#select")
-          item.setAttribute("data-satis-dropdown-item-value", option.value)
-          item.setAttribute("data-satis-dropdown-item-text", option.text)
-          item.classList.add("cursor-pointer", "px-4", "py-2", "hover:bg-primary-200", "hover:font-medium", "text-sm")
-          item.innerHTML = option.text
+          item.innerHTML = `
```

Comment:
we should maybe do this with templates in the view, that's what we use them for.